tools/DoctorMyPass.py

```
@@ -0,0 +1,80 @@
+    print(f"Current Working Directory: {info['cwd']}")
+    print("\n\U0001F4C2 sys.path (where Python looks for modules):")
+    for p in info['sys_path']:
+        print(f" - {p}")
+    print("\n\U0001F4DA Importable Modules (minimal check):")
+    for mod in info['modules']:
+        print(f" - {mod}")
+
+
+def save_report(info, filename):
+    try:
+        with open(filename, "w") as f:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)